sentry-tower/src/lib.rs

```diff
@@ -326,3 +326,22 @@ impl<S, Request> NewSentryService<S, Request> {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use std::rc::Rc;
+
+    #[allow(dead_code)]
```

Comment:
why would this `allow` be needed?

Reply:
because it's not a `#[test]`, so it's not run. 

( and the only thing we want is to check if `Sync` is correct, so it should just compile)

Reply:
You can just turn the test functions into proper `#[test]`s as well. You are right that they are essentially noops at runtime though.

Reply:
done, ( I amended & force-pushed, didn't want to create another commit for this)

Reply:
those details don’t matter as this repo is configured to use squash&merge anyway :-)